src/screens/Auth/Login/Login.js

```diff
@@ -32,34 +30,25 @@ const theme = {
   },
 };
 
-function Login() {
+function Login()
+{
   const dispatch = useDispatch();
   const [loading, setLoading] = useState(false);
   const [isPasswordSecure, setIsPasswordSecure] = useState(true);
+  const [responseError, setResponseError] = useState();
 
-  const onLoginPress = async (data) => {
+  const onLoginPress = async (data) =>
+  {
+    setResponseError(null);
     setLoading(true);
-    const dbConfig = await getEncryptedData(
-      `dbCredentials_${data?.password}_${data?.email.replace('@', '')}`
-    );
-    if (dbConfig) {
-      dispatch(login(dbConfig, { email: data?.email, password: data?.password }));
-    } else {
-      new API()
-        .login({ email: data?.email, password: data?.password })
-        .then((response) => {
-          setLoading(false);
-          if (response.error) {
-            console.log ("Login_page_error : ", response.error)
-            return;
-          }
-          dispatch(login(response, data));
-        })
-        .catch((error) => {
-          setLoading(false);
-          console.error(error);
-        });
+    const response = await fetchAuthCredentials({ username: data?.email, password: data?.password })
+    setLoading(false);
+    
+    if (response.error) {
+      setResponseError(response.error);
+      return;
```

Comment:
Wouldn't we set the loading to false after having the response error so we remove the loader only when the message is available?

```suggestion    
    if (response.error) {
      setResponseError(response.error);
      setLoading(false);
      return;
    }
    setLoading(false);
```

Reply:
Same comment for the other error handleling like in onSignUp

Reply:
The Line 45 has the setLoading(false); 

Also, the line  47  has duplicated the code of line  49: 

 if (response.error) {
      setResponseError(response.error);

I seems as a merge error

Reply:
It seems like a merge error since we have 

- line 45 : still has setLoading(false);
- line 47 has the duplicated code of line 49